pkg/auth/dependency/newinteraction/intents/authenticate.go

```diff
@@ -91,9 +98,10 @@ func (i *IntentAuthenticate) DeriveEdgesForNode(ctx *newinteraction.Context, gra
 		}
 
 		return []newinteraction.Edge{
-			&nodes.EdgeCreateIdentityBegin{RequestedIdentity: selectIdentity.RequestedIdentity},
+			&nodes.EdgeCreateIdentityEnd{
```

Comment:
I suppose using `NodeCreateIdentityBegin` to pair with `EdgeCreateIdentityEnd` would be cleaner. Let's review it later after implementing add identity page.